test/unit/internal/test_emerge_planner.py

```diff
@@ -611,3 +614,25 @@ def assert_same_part_definitions(self, emerge_plan, expected_part_defs):
                     self.assertIs(emerge_part_def.copy_source, expected_part_def.copy_source)
                 self.assertEqual(emerge_part_def.relative_offset, expected_part_def.relative_offset)
                 self.assertEqual(emerge_part_def.length, expected_part_def.length)
+
+
+class TestEmergePlannerHugeFile(TestEmergePlanner):
+    def setUp(self):
+        super().setUp()
+        self.planner = self._get_emerge_planner()
+
+    def _get_emerge_planner(self):
+        return EmergePlanner(
+            min_part_size=self.min_size,
+            recommended_upload_part_size=self.recommended_size,
+            max_part_size=self.max_size,
+        )
```

Comment:
`_get_emerge_planner` the same as `TestEmergePlanner._get_emerge_planner`, why do you duplicate this?
And why do you call `super().setUp(); self.planner = ...` if it's exactly the same as `TestEmergePlanner.setUp()`?

Reply:
Good catch. I planned to use it to decrease the max part count but in he end I didn't have to.